conf/app.routes

```diff
@@ -1,4 +1,10 @@
-###########################     Service pages               ########################
+#################       Cis details pages   ########################################
+
+GET       /:taxYear/contractor-details                                              controllers.ContractorDetailsController.show(taxYear: Int, contractor: Option[String])
```

Comment:
this section can be moved to below the summary pages with the rest of the EOY pages